project/build.scala

```diff
@@ -0,0 +1,83 @@
+import sbt.Keys._
+import sbt._
+
```

Comment:
I thought that binding was being split out to its own repository though, so you shouldn't need the subprojects anymore, right?